content/en/docs/02/03_restore.md

```diff
@@ -4,3 +4,56 @@ weight: 23
 sectionnumber: 2.3
 ---
 
+## Task {{% param sectionnumber %}}.1: Restoring user workload backups
+
+In the backup lab you created a scheduled backup of all resources in namespace `uptime-app-prod`. Now we will test the restore process.
+
+Go ahead and delete the namespace `uptime-app-prod`:
+
+```bash
+oc delete ns uptime-app-prod
```

Comment:
I have to test the behavior.
If project specific config like annotations or labels are not backed up (because not k8s native), then we need to rethink the use of Velero anyway.

Reply:
Deleting the namespace deletes the project anyway, so I don't see a difference.

Reply:
Annotations are preserved, changed to `delete project`.